src/Build/Logging/BinaryLogger/ProjectImportsCollector.cs

```diff
@@ -165,6 +169,15 @@ private void AddFileCore(string filePath)
             using FileStream content = new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.Read | FileShare.Delete);
             AddFileData(filePath, content, null);
         }
+        private void InvokeFileIOErrorEvent( string filePath,string message)
```

Comment:
nit: add an empty line before the method signature

Reply:
nit: and the same spacing comment in the method args

Reply:
Extra points - find out if there is .editorconfig setting we can/should turn on to catch this (that at the same time doesn't lead to bunch of errors elsewhere in the code)